lib/phoenix/code_reloader/server.ex

```diff
@@ -48,7 +48,7 @@ defmodule Phoenix.CodeReloader.Server do
             Mix.Project.build_structure()
           else
             Logger.warning(
-              "Phoenix is unable to create symlinks. Phoenix' code reloader will run " <>
+              "Phoenix is unable to create symlinks. Phoenix's code reloader will run " <>
```

Comment:
I'm not a native speaker, but do you really add an `s` if the word already ends in an s-like sound, which Phoenix does? `ˈfiːnɪks`

Reply:
The internet says yes.